kimera_semantics/src/semantic_integrator_base.cpp

```diff
@@ -206,6 +215,9 @@ SemanticVoxel* SemanticIntegratorBase::allocateStorageAndGetSemanticVoxelPtr(
   const vxb::BlockIndex& block_idx = vxb::getBlockIndexFromGlobalVoxelIndex(
       global_voxel_idx, semantic_voxels_per_side_inv_);
 
+  const vxb::VoxelIndex local_voxel_idx = vxb::getLocalFromGlobalVoxelIndex(
+          global_voxel_idx, semantic_voxels_per_side_);
```

Comment:
Is there a reason why this is now here? It was some lines below before, right?

Reply:
No specific reason, I wanted to use that local index for initialization, but changed my implementation later. 
Rearranged to original place, thank you fro spotting